src/bindings/haskell/README.md

```diff
@@ -40,13 +40,17 @@ which contains the current list of haskell packages by invoking `cabal update`,
 is not done automatically. Then first ensure the following programs are available on
 the PATH:
 
-- c2hs (tested with 0.28.2)
+- c2hs (tested with 0.28.2 and 0.28.5)
+  c2hs depends on the language-c library and is usually compiled from scratch when installed
```

Comment:
Maybe put the error message here so that troubleshooting is easier.

Reply:
I don't think thats helpfull as you need to compile with -v2 to see the full error message that is not the default. But i think it won't really be that much of an issue due to the language-c update on hackage.